testing/TestHarness/TestHarness.Shared/Ext/Validation/ValidationOnePage.xaml

```diff
@@ -0,0 +1,14 @@
+﻿<Page
+    x:Class="TestHarness.Ext.Navigation.Validation.ValidationOnePage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:TestHarness.Ext.Navigation.Validation"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+    <Grid>
+
```

Comment:
We should include the results of the validators on the UI - then we can actually run this as part of the UI Tests